accord-core/src/test/java/accord/impl/basic/DelayedCommandStores.java

```diff
@@ -266,22 +265,14 @@ public DelayedSafeStore(DelayedCommandStore commandStore, RangesForEpoch ranges,
         @Override
         public void postExecute()
         {
-            if (context instanceof Message)
```

Comment:
not related to this patch... I added this logic to detect mutations outside of types with side effects, but really we should check that we can still load the command that we are trying to write...  this makes sure that we don't bias towards only checking for txn we re-load